index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/7b47055909.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Raleway:200&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+<!-- <div class="container"> -->
+    <div class="back-ground-image">
+        <div class="nav">

```

Comment:
[OPTIONAL]
Don't forget to use [Semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp). It is better to use something like `nav` i. nthis case.
This requirement can be seen in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) for mpathwright)